packages/ui/src/components/Toast/Toast.tsx

```diff
@@ -2,23 +2,23 @@ import { PropsWithChildren, ReactNode } from "react";
 import { toast as sonnerToast, Toaster } from "sonner";
 import { cva } from "class-variance-authority";
 
-import { Icon } from "../Icon";
-import { Button } from "../Button";
+import { IconButton, Button, Icon, IconName } from "@/components";
 
 export const toastStyles = cva(
   [
     "flex items-center",
-    "w-fit px-4 py-3 space-x-4",
-    "bg-surface-surface-0 border shadow-4 rounded-20",
-    "text-md font-bold",
+    "w-fit px-4 py-3.5 space-x-4",
+    "bg-surface-surface-white-smoke-1 border shadow-6 rounded-20",
+    "text-text-low-em",
   ],
   {
     variants: {
       colorScheme: {
-        default: ["border-outline-low-em"],
-        error: ["border-outline-danger-med-em"],
-        success: ["border-outline-success-med-em"],
-        warning: ["border-outline-warning-med-em"],
+        default: ["border-outline-base-em"],
```

Comment:
Maybe call it main?

Reply:
not related with variant styles, it's related to the `base/low/med/high` terminology that Akash used.